docs/en/ingest-management/elastic-agent/ingest-pipeline-kubernetes.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[ingest-pipeline-kubernetes]]
+= Using Custom Ingest Pipeline with {k8s} Integration
```

Comment:
```suggestion
= Using a custom ingest pipeline with the {k8s} Integration
```
Sorry I missed this before. We usually capitalize only the first word, and proper names.